### DISCUSSION TITLE
STI

### DISCUSSION THREAD

**Discussion by GIOLP** [Ifsttar/I-Simpa#353] [General]
Hi
I'd like tò learn using I-Simpa in architectural acoustics field. Only two questions:
- is  this soft  able tò simulate all room acoustics parameters (C80,D50,Ts,G,Sti,Iacc...)? 
- it's possibile tò import 3d model from SketchUp soft like make Odeon, Catt...without any plugins?
Thank you in advance,
Giovanni

**Comment by nicolas-f** [comment]
Hi,

Thanks for your interest in I-Simpa.

Supported acoustics parameters are listed in the documentation here:
https://i-simpa-wiki.readthedocs.io/fr/latest/room_acoustics_parameters.html

You can import 3D models in STL format
There is a tutorial that explain that part:
https://i-simpa-wiki.readthedocs.io/fr/latest/tutorial_Elmia_hall.html
Please read this part too:
https://i-simpa-wiki.readthedocs.io/fr/latest/import_file_recommandations.html

regards,
Nicolas
